app/uk/gov/hmrc/customs/declaration/services/CustomsAuthService.scala

```diff
@@ -72,9 +72,9 @@ class CustomsAuthService @Inject()(override val authConnector: AuthConnector,
   /*
   Wrapper around HMRC authentication library authorised function for CSP authentication
   */
```

Comment:
How about a type alias like `type RequestRetrievals = Boolean` used here, like `IsCsp` for clarity?

Reply:
Done